### ISSUE TITLE
Show what's updated

### ISSUE THREAD

**Issue by PHELAT** [cesarferreira/drone#6]
```
Already up-to-date.

Already up-to-date.

Updated successfully
```

that's what drone printed out while updating my project, it would be nice if drone could also print what library is "Updated successfully" and what is "Already up-to-date".

**Comment by cesarferreira** [comment]
You're right, this is the https://github.com/cesarferreira/drone-hive pulling the latest changes :) 
I'll put a message or hide it :) thanks for pointing it out
